applications/pelion_client/src/modules/net_ot.cpp

```diff
@@ -0,0 +1,213 @@
+/*
```

Comment:
Why is this a cpp file? OT exposes C API.

Reply:
Don't know. Probably nobody checked it works for C and since OT lib is in C++ the file appeared as cpp. Will try to change the extension.

Reply:
nope - it fails as we refer to ot_context. so no change here

Reply:
That's a bit weird ot_context is an issue here - it's still C.

Reply:
That's the static_cast.... will change and retry

Reply:
Fixed